cmd/notation/policy/show.go

```diff
@@ -0,0 +1,47 @@
+package policy
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/spf13/cobra"
+)
+
+type showOpts struct {
+}
+
+func showCmd() *cobra.Command {
+	var opts showOpts
+	command := &cobra.Command{
+		Use:   "show [flags]",
+		Short: "[Preview] Show trust policy configuration",
+		Long: `[Preview] Show trust policy configuration.
+
+** This command is in preview and under development. **
+
+Example - Show current trust policy configuration:
+  notation policy show
+
+Example - Save current trust policy configuration to a file:
+  notation policy show > my_policy.json
+`,
+		Args: cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runShow(cmd, opts)
+		},
+	}
+	return command
+}
+
+func runShow(command *cobra.Command, opts showOpts) error {
+	// read policy json contents
+	policyJson, err := os.ReadFile(filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy))
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "no trust policy configuration found, you may import one via `notation policy import <path-to-policy.json>`")
+		return fmt.Errorf("failed to read trust policy configuration: %w", err)
+	}
```

Comment:
I thought this is what we discussed to do?
```suggestion
	policyDoc, err := trustpolicy.LoadDocument()
	if err != nil {
		fmt.Fprintln(os.Stderr, "failed to load trust policy configuration, you may import one via `notation policy import <path-to-policy.json>`")
		return err
	}
	if err := policyDoc.Validate(); err != nil {
		fmt.Fprintln(os.Stderr, "existing trust policy configuration is invalid, you may import one via `notation policy import <path-to-policy.json>`")
		return err
	}
```

Reply:
Here's one of the reasons that we need `Validate()` here:
Let's say the user wants to add a new trust policy statement to the existing trust policy document. If the existing trust policy doc is not valid, and `notation policy show` does not report it, then the user would just add their new statement into the original invalid document and call `notation policy import` to apply the change. And since `notation policy import` calls `Validate()` on the new document, it would not surprisingly fail. Only until this point, the user will realize the original policy doc is invalid and all their previous work needs to be redone. Thus, explicitly displaying the existing trust policy is malformed before the user make any change on it is necessary.

Reply:
`fmt.Fprintf` does not support error-wrapping directive `%w`, changed it to `%v`